regression/rt_gk_mdpx_cart_3x2v_p1.c

```diff
@@ -434,11 +434,8 @@ main(int argc, char **argv)
     .collisions =  {
       .collision_id = GKYL_LBO_COLLISIONS,
       .normNu = true,
-      .self_nu_fac = norm_nu_func(ctx.nuFrac, ctx.n0, ctx.n0, ctx.massElc, ctx.massElc, ctx.chargeElc, ctx.chargeElc, ctx.Te, ctx.Te),
-      .cross_nu_fac = {
-        norm_nu_func(ctx.nuFrac, ctx.n0, ctx.n0, ctx.massElc, ctx.massAr, ctx.chargeElc, ctx.chargeIon, ctx.Te, ctx.TAr)
-      },
-      .bmag_mid = ctx.B0,
+      .n_ref = ctx.n0,
+      .T_ref = ctx.Te,
```

Comment:
@akashukla likewise, since these are examples for others, please comment on these lines and explain what n_ref and T_ref are.